sockeye/arguments.py

```diff
@@ -248,7 +285,10 @@ def add_model_parameters(params):
     model_params.add_argument('--attention-coverage-num-hidden',
                               type=int,
                               default=1,
-                              help="Number of hidden units for coverage vectors. Default: %(default)s")
+                              help="Number of hidden units for coverage vectors. Default: %(default)s.")
+    model_params.add_argument('--attention-mhdot-heads',
```

Comment:
ahh ok, got it. Could you maybe add the word `self` to the other attention argument or make it clear in the help string that the other argument is about self-attention. It all makes sense now, but initially looking at the parameters I didn't realize that one param was for the attention mechanism from enc to dec and the other for self-attention.  Maybe:
```
model_params.add_argument('--transformer-attention-heads',
                              type=int_greater_or_equal(1),
                              default=8,
                              help='Number of heads for all self-attention when using transformer layers. Default: %(default)s.')
```